### DISCUSSION TITLE
Frameworks `useMachine` return

### DISCUSSION THREAD

**Discussion by anubra266** [statelyai/xstate#4721] [General] [answered]
I'm curious; why does the react wrapper return a tuple and the other frameworks return an object. This doesn't feel consistent.

**Comment by Andarist** [comment, marked as answer]
The intention was to match React/Vue/Svelte/Solid conventions - not to have it consistent between our packages.

**Reply by anubra266** [reply]
Ok...I understand. Thanks @Andarist
